README.md

```diff
@@ -30,8 +33,7 @@ ${{ steps.semantic_release_info.outputs.<variable name> }}
 In this example I get changelog for the future release and add a comment to the pr with it.  
```

Comment:
GitHub doesn't let me comment on the correct line but in the previous section, the ending should be the beginning: 

## Output variables

After the action is completed, you can access the output variables using this pattern: 

```
${{ steps.semantic_release_info.outputs.<variable name> }}
```

- list of variables....